### PR TITLE
release: show the right opam-publish command to run

### PR DESCRIPTION
**Pull request by kit-ty-kate**
Requires https://github.com/ocaml-opam/opam-publish/pull/161

### CODE REVIEW

File: release/readme.md [hunk 1]
```diff
@@ -67,7 +67,8 @@
 ## Publish the release
 
 * add hashes in `install.sh` (and check signatures)
-* publish opam packages in opam-repository (and add `flags: avoid-version` and `available: opam-version >= "2.1.0"` to each packages if this is not a stable version)
+* `opam publish --exclude=opam,opam-admin https://github.com/ocaml/opam/archive/refs/tags/<tag>.tar.gz`
+  (and add `flags: avoid-version` and `available: opam-version >= "2.1.0"` to each packages if this is not a stable version)
```
Comment (dra27) [inline release/readme.md:71]: Is that worth having a feature in opam-publish for? (`--pre-release` or some such?)

**Review by dra27 — APPROVED** [review]
Pre-approval for when the opam-publish PR is merged and released 😊

File: release/readme.md (hunk 1 above)
Comment (kit-ty-kate) [reply release/readme.md:71]: Yes, definitely. I did think about it but forgot to open a ticket. Done in https://github.com/ocaml-opam/opam-publish/issues/163

**Review by rjbou — APPROVED** [review]
Thanks!

**Pull request merged**